python-package/xgboost/training.py

```diff
@@ -369,7 +369,7 @@ def aggcv(rlist):
 def cv(params, dtrain, num_boost_round=10, nfold=3, stratified=False, folds=None,
        metrics=(), obj=None, feval=None, maximize=False, early_stopping_rounds=None,
        fpreproc=None, as_pandas=True, verbose_eval=None, show_stdv=True,
-       seed=0, callbacks=None, shuffle=True):
+       seed=0, callbacks=None, shuffle=True, as_modin=False):
```

Comment:
Can we remove changes to cv for now?  I don't like this function and seeking for external implementation, or at least a better re-implementation.

Reply:
yes. But seems I should also delete related test case for Modin, is it ok?

Reply:
That's fine.

Reply:
done :)